src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage{
+    String newPlaylistName1 = "Test22";
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //Web Elements
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By playlistNameField = By.cssSelector("name='name'");
+    By renamePlaylistSuccessMsg = By.cssSelector("div.success.show");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```